### ISSUE TITLE
Very bad and glitchy mod configurations

### ISSUE THREAD

**Issue by Betestav** [octarine-noise/BetterFoliage#445]
I do love this mod for its visuals and great natural elements aditions, but if there are some very frustrating problems about it , it's certainly the lack of configuration and optimisations the mod has. Let me explain. 
Yes, you can change and ajust most of the settings of this mod whitout any problem, but im gonna demistify the problems and the settings that isn't the case. 

First of all, disabling the rendering of extra leaves on trees. If i do that and load in a world, the game instantly crashes as soon as there are leaves to render. we want those leaves to just not appear in the trees, not crash the game!? (this hapens because of optifine, the hd u g8 version with forge 36.2.10 , minecraft 1.16.5)

Second, Terrible optimisation. I've discovered that in the past, there was options such as "lod" or "render distance" for those extremely dense leaves to render. Because yes, even a rtx 3070 ti struggles to render alot of leaves like that in a 20 chunk radius in a dark oak biome. It's ridiculous, why can't we set a value on how far those leaves can appear? My frame rate takes serious tackles everytime i turn my camera at some regions when i play very often. Why removing such feature??

Not to mention that the "hide internal leaves" setting does NOT work AT ALL, and it sucks, the extra leaves are still rendered inside the trees, ALSO, it overides  the "smart leaves" detail option from optifine so the vanilla leaves block (the faces) inside trees are also not rendered. This result in alot of useless geometry that the gpu has to render, and we can't even see them. Put all these on top of another and it will create alot of useless lag. If they are fixed, it won't even affect visuals.

This negligence of just not making the mod working as intended, Nothing justified this lack of care.

 So here is a recapitulation of the problems:
1: To many leaves rendered inside trees even tho we don't even see them (hide internal leaves doesn't do shit)
2: no render distance setting for the extra leaves
3: trees can't be hollow with the "smart leaves" setting on from optifine, mod overides the setting and puts it in fancy, resulting in extra useless geometry that has to be rendered, thus creating unnecesary lag.
4: game crashing when we just simply want to disable the extra leaves rendering. (this is because of optifine, the hd u g8 with forge 36.2.10)
Basically it turns arround of leaves.

My pc specs:
i7 13700kf
Rtx 3070 Ti
32gb 5600mhz DDR5
asrock Z690 Steel Legend D5 mb
pc temps are well good on all devices.

Considering my pc specs, It's ridiculous to think it could lag. I USE SHADERS WITH THE MOD, THAT IS WHY I COMPLAIN.
